src/pages/books.js

```diff
@@ -34,15 +38,30 @@ const BooksPage = ({ data }) => (
       <div className="bookshelf">
         <div className="book">
           <h1>January</h1>
-          <Img fixed={data.atomicHabits.childImageSharp.fixed} />
+          <a href={LINK_ATOMIC_HABITS} target="_blank">
+            <Img fixed={data.atomicHabits.childImageSharp.fixed} />
+          </a>
+          <a href={LINK_ATOMIC_HABITS} target="_blank">
+            <button className="button button-border">Details</button>
```

Comment:
This is great thanks! You don't need to wrap a button in a link - it's bad for a11y. Just style a link element like a button. You can add `className='link-button'` or something similar and use the same styling.

Reply:
My bad! Sure, I will change and push it today evening.
Thanks for your review.

Reply:
I am sorry for the delay. My computer didn't work the last three days, so I couldn't proceed.
Will finish it until tomorrow.
Thanks and sorry again.